paper/paper.md

```diff
@@ -0,0 +1,143 @@
+      index: 1
+    - name: University of Washington, Seattle
+      index: 2
+    - name: Washington University, St. Louis
+      index: 3
+    - name: Independent Contributor
+      index: 4
+
+date: 29 October 2021
+bibliography: paper.bib
+---
+
+# Summary
+
+Cities, counties, and states throughout the USA are bound by law to archive recordings of public meetings. Most local governments comply with these laws by posting documents, audio, or video recordings online. As there is no set standard for municipal data archives however, parsing and processing such data is typically time consuming and highly dependent on each municipality. Council Data Project (CDP) is a set of open-source tools that improve the accessibility of local government data by systematically collecting, transforming, and re-publishing this data to the web. The data republished by CDP is packaged and presented within a searchable web application that vastly simplifies the process of finding specific information within the archived data. We envision this project being used by a variety of groups including civic technologists hoping to promote government transparency, researchers focused on public policy, natural language processing, machine learning, or information retrieval and discovery, and many others.
+
+# Statement of Need
+
+Comparative research into municipal governance in the USA is often prohibitively difficult due to a broad federal system where states, counties, and cities divide legislative powers differently. This has contributed to the lack of large-scale quantitative studies of municipal government, and impeded necessary research into effective procedural elements of administrative and legislative processes [@Trounstine:2009]. Council Data Project enables large-scale quantitative studies by generating standardized municipal governance corpora - including legislative voting records, timestamped transcripts, and full legislative matter attachments (related reports, presentations, amendments, etc.).
+
+## Related Work
+
+Work in extracting and repackaging government data into machine-readable and experiment ready datasets has historically happened in fields with highly structured data, such as meteorology [@Sparks2017] and legal review and monitoring [@courtlistener]. Notably, there has been prior work in extracting and repackaging municipal government data with [Councilmatic](https://github.com/codeforamerica/councilmatic) [@councilmatic]. However, this work largely aimed to make municipal data more accessible to a general public, and didn't add any specific data processing to expand the research capabilities of the produced dataset. Recent advances in natural language processing have made it possible to conduct large-scale transcript-based studies on the effects of gender, ideology, and seniority in Supreme Court oral argument [@jacobi2017] and the effects that information communication technology has on civic participation [@einstein2021].
```

Comment:
"...this work largely aimed to make municipal..." --> "...this work **is** largely aimed to make municipal..."

Reply:
"...and **didn't** add any specific data..." --> "...and **doesn't** add any specific data..."

(Unless of course, Councilmatic work has stopped in which case the previous comment should use **was** not **is**).

Reply:
"...Supreme Court oral **argument**..." --> "...Supreme Court oral **arguments**..." ??

Reply:
Accepted on councilmatic stuff. legal argument is the formal term. idk why but that is the norm.